networkit/cpp/coarsening/ParallelPartitionCoarsening.cpp

```diff
@@ -13,124 +15,97 @@
 #include <networkit/coarsening/ParallelPartitionCoarsening.hpp>
 #include <networkit/graph/GraphBuilder.hpp>
 
+#include <iostream>
```

Comment:
This is not needed anymore right?

Reply:
Correct. Is there a way to remove it without re-triggering the CI?

Reply:
No, CI is triggered after every change. It's quite annoying that it needs to be done manually, we should probably change this setting.